cmd/README.md

```diff
@@ -187,6 +187,10 @@ where `test/test.js` is the path of the file.
 
 You have the flexibility to write tests in quite a variety of ways, open to you to explore.
 
+## Running your application
+For the purpose of *testing* APIs or prototyping user interfaces, you can run your DNA using `hc run`.
+This will start the application and open a WebSocket on port `8888` or any other port specified by the `-p`/`--port` option. The `-b`/`--package` flag will build your DNA before running it. By default, none of the data your application is putting on the SourceChain. To persist data onto the file system, use the `--persist` flag.
```

Comment:
```suggestion
This will start the application and open a WebSocket on port `8888` or any other port specified by the `-p`/`--port` option. The `-b`/`--package` flag will build your DNA before running it. By default, none of the data your application is putting on the source chain gets persisted. To persist data onto the file system, use the `--persist` flag.
```